tests/test-workflows.yml

```diff
@@ -956,7 +956,7 @@
         - "1\t1,1.5,tomato,0.potato,1,000"
         - "2\t2,fish"
         - "3\t3,quote"
-        - "4\t4.0"
+        - "4\t4,4.0"
```

Comment:
This was wrong before since it was erroneously converting 4 into 4.0 and now it correctly has both.